### ISSUE TITLE
Vitamio is getting crash for some of the rtmp url's

### ISSUE THREAD

**Issue by saveme925** [yixia/VitamioBundle#238]
Hi,
I am using your product into my Android application, Its working fine for most of the url's but fails at some place. I am not sure what is the exact problem, I have tested with your sample code, still getting crash for that. Could you please help me to rectify the problem.
Here is the URL
rtmp://oregon.owvideo.greenjobinterview.net/oneway/_definst_/intro/722/6324_1382042895157

**Comment by rockchip** [comment]
From what I can tell, it seems like you are missing rtmp parameters to make that stream work. Tried it using ffplay and got the following errors:

<pre>
ffplay -i rtmp://oregon.owvideo.greenjobinterview.net/oneway/definst/intro/722/6324_138204289515
7
ffplay version N-68129-g928322c Copyright (c) 2003-2014 the FFmpeg developers
  built on Dec  1 2014 02:39:44 with gcc 4.9.2 (GCC)
  configuration: --disable-static --enable-shared --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzl
ib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-l
ibcaca --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libope
ncore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libs
oxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-aacenc --enable-libvo-amrwbenc --ena
ble-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxv
id --enable-decklink --enable-zlib
  libavutil      54. 15.100 / 54. 15.100
  libavcodec     56. 13.100 / 56. 13.100
  libavformat    56. 15.101 / 56. 15.101
  libavdevice    56.  3.100 / 56.  3.100
  libavfilter     5.  2.103 /  5.  2.103
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  1.100 /  1.  1.100
  libpostproc    53.  3.100 / 53.  3.100
Closing connection: NetStream.Play.StreamNotFound sq=    0B f=0/0
rtmp://oregon.owvideo.greenjobinterview.net/oneway/definst/intro/722/6324_1382042895157: Unknown error occurred
    nan    :  0.000 fd=   0 aq=    0KB vq=    0KB sq=    0B f=0/0
</pre>

Try passing the pageurl and swfurl in the AVOptions.
https://github.com/yixia/VitamioBundle/wiki/Use-FFmpeg-AVOption-in-Vitamio-4.1.0

**Comment by saveme925** [comment]
This is what I am doing 
                        uri = Uri.parse(path);
			mVideoView.setVideoURI(uri);
			mVideoView.requestFocus();
			mVideoView.setOnInfoListener(this);
			mVideoView.setOnBufferingUpdateListener(this);
			mVideoView
					.setOnPreparedListener(new MediaPlayer.OnPreparedListener() {
						@Override
						public void onPrepared(MediaPlayer mediaPlayer) {
							
							mediaPlayer.setPlaybackSpeed(1.0f);
						}
					});

**Comment by rockchip** [comment]
Well just by adding the <pre> -rtmp_live 1 </pre> parameter I managed to get a handshake error. I'm assuming its protected content? I can't tell you why the iOS library is different.
<pre>
ffplay -i rtmp://oregon.owvideo.greenjobinterview.net/oneway/definst/intro/722/6324_138204289515
7 -rtmp_live 1
ffplay version N-68129-g928322c Copyright (c) 2003-2014 the FFmpeg developers
  built on Dec  1 2014 02:39:44 with gcc 4.9.2 (GCC)
  configuration: --disable-static --enable-shared --enable-gpl --enable-version3 --disable-w32threads --enable-avisynth --enable-bzl
ib --enable-fontconfig --enable-frei0r --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libbs2b --enable-l
ibcaca --enable-libfreetype --enable-libgme --enable-libgsm --enable-libilbc --enable-libmodplug --enable-libmp3lame --enable-libope
ncore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-librtmp --enable-libschroedinger --enable-libs
oxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvo-aacenc --enable-libvo-amrwbenc --ena
ble-libvorbis --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxv
id --enable-decklink --enable-zlib
  libavutil      54. 15.100 / 54. 15.100
  libavcodec     56. 13.100 / 56. 13.100
  libavformat    56. 15.101 / 56. 15.101
  libavdevice    56.  3.100 / 56.  3.100
  libavfilter     5.  2.103 /  5.  2.103
  libswscale      3.  1.101 /  3.  1.101
  libswresample   1.  1.100 /  1.  1.100
  libpostproc    53.  3.100 / 53.  3.100
HandShake: client signature does not match!   0KB sq=    0B f=0/0
rtmp server sent error=   0 aq=    0KB vq=    0KB sq=    0B f=0/0
RTMP_ReadPacket, failed to read RTMP packet headersq=    0B f=0/0
rtmp://oregon.owvideo.greenjobinterview.net/oneway/definst/intro/722/6324_1382042895157: Invalid data found when processing input
</pre>

If it's a swf rtmp feed you will definetly need the AVOptions set.

**Comment by saveme925** [comment]
Yes I have tried with the AVOptions set but still getting same message and then crash. What could be the cause of this problem or is there any working solution for this because everything is working fine when I use the same in iOS. I have tried with Vitamio sample code, but didn't get any luck on it.

**Comment by saveme925** [comment]
The message I got is 
Fatal signal 11 (SIGSEGV) at 0x54e2a000 (code=1), thread 6131 (ov.vitamio.demo)
and my url is
rtmp://oregon.owvideo.greenjobinterview.net/oneway/_definst_/intro/361/2329_1422050345864
